releases/release-1.31/release-notes/maps/pr-125225-map.yaml

```
@@ -0,0 +1,5 @@
+pr: 125225
+releasenote:
+  text: The Service LoadBalancer controller was not correctly considering the service.Status
+    new IPMode field and excluding the Ports when comparing if the status has changed,
+    causing that changes in these fields may not update the service.Status correctly
```

Assuming this is a bug fix:
```suggestion
  text: Fixed an issue where the Service LoadBalancer controller was not correctly considering the service.Status
    new IPMode field and excluding the Ports when checking if the status was changed,
    resulting in the changed field potentially not to update the service.Status correctly.
```